src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Layout/Header.razor.css

```diff
@@ -0,0 +1,20 @@
+.navbar-brand {
+    height: 3.5rem;
+    font-size: 1.1rem;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    margin: 0;
+    background-color: var(--bl-blazor-brand-color1);
+}
+
+    .navbar-brand, .navbar-brand:hover, .navbar-brand:visited {
```

Comment:
Since you're making improvements, is it too soon to use native css nesting?

```css
.navbar-brand {
    height: 3.5rem;
    font-size: 1.1rem;
    width: 100%;
    display: flex;
    align-items: center;
    margin: 0;
    background-color: var(--bl-blazor-brand-color1);
    
    &, &:hover, &:visited {
       color: var(--bl-blazor-brand-contrast-color);
    }
    
    @media (width >= 641px) {
       width: 250px;
       margin-right: 1rem;
    }
}
```

Reply:
Sure, I didn't realize it was supported in all browsers. Looks like it is according to "can I use"

Reply:
Updates have been pushed